bpf/lib/ipv6.h

```diff
@@ -237,20 +237,6 @@ static inline void ipv6_set_lxc_id(union v6addr *addr, __u16 lxc_id)
 	addr->p4 = htonl(p4);
 }
 
-static inline __u16 ipv6_derive_state(const union v6addr *addr)
-{
-	return (ntohl(addr->p4) >> 16);
-}
-
-/* FIXME: rewrite this to avoid byte order conversion */
-static inline void ipv6_set_state(union v6addr *addr, __u16 state)
-{
-	__u32 p4 = ntohl(addr->p4);
-	p4 &= 0xFFFF;
-	p4 |= (state << 16);
-	addr->p4 = htonl(p4);
-}
-
```

Comment:
I removed them because the byte order conversion  should not happen in the fast path at all. So when we do the encap part, let's add it properly with the minimal instructions required.